src/test/java/api/get/GetActionsOfBoardTest.java

```diff
@@ -0,0 +1,37 @@
+package api.get;
+
+import api.BaseTest;
+import jdk.jfr.Description;
+import org.apache.http.HttpStatus;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import java.util.Map;
+
+import static io.restassured.module.jsv.JsonSchemaValidator.matchesJsonSchemaInClasspath;
+
+
+public class GetActionsOfBoardTest extends BaseTest {
+
+    private static String ID_BOARD;
+
+    @BeforeMethod
+    public void createBoard() {
+        ID_BOARD = boardRestTestClient.createNewBoard(Map.of("name", "Test board")).getId();
+    }
+
+    @Test
+    @Description("Getting information about action on the boards")
+    public void GetActionsOfBoard() {
+        actionsRestTestClient.getActionOfBoard( ID_BOARD)
+                .assertThat()
+                .statusCode(HttpStatus.SC_OK)
+                .body(matchesJsonSchemaInClasspath("ResponseSchemaDirectory/schema.json"));
+    }
+
+    @AfterMethod
+    public void delete() {
+        boardRestTestClient.deleteBoardIfExist(ID_BOARD);
+    }
+}
```

Comment:
We have same code for setup and teardown in other test classes whose test methods touch the board functionality. Why can't we put these test methods in one class? Or extract these methods into some abstract class that these test classes can extend? This way, we can avoid duplication of @AfterMethod and @BeforeMethod.